x-pack/plugin/build.gradle

```diff
@@ -119,6 +119,4 @@ validateRestSpec {
   ignore 'autoscaling.delete_autoscaling_policy.json'
   ignore 'autoscaling.get_autoscaling_policy.json'
   ignore 'autoscaling.put_autoscaling_policy.json'
-  ignore 'ml.validate.json'
```

Comment:
I think these are stable APIs, so would clash with the previous comment about allowing `null` only for beta and experimental APIs. Perhaps we can link to ML jobs documentation for these specs for now? For example, 

https://www.elastic.co/guide/en/machine-learning/current/ml-jobs.html